lib/workflowmgr/slurmbatchsystem.rb

```diff
@@ -300,8 +300,9 @@ def submit(task)
 
           queued_jobs,errors,exit_status=WorkflowMgr.run4("squeue -u #{username} -M all -t all -O jobid:40,comment:32", 45)
 
-          # Raise SchedulerDown if the command failed
-          raise WorkflowMgr::SchedulerDown,errors unless exit_status==0
+          # Don't raise SchedulerDown if the command failed, otherwise
```

Comment:
This squeue command does not seems subject to the bug you found since it does not use the --jobs option. In my testing it works just fine whether there are jobs to list or not.  If this command fails, it should be because there is a problem with the batch system, not due to jobs transitioning to sacct.  Right?